index.html

```diff
@@ -137,6 +143,25 @@ <h4>
           </dl>
         </section>
         <section>
+          <h4>
+            <dfn>canShareFiles()</dfn> method
+          </h4>
+          <p>
+            Each user agent should have an `allowlist` of permitted MIME types
+            and file extensions or file names. These may contain wildcards, for
+            example `*.txt` or `text/*`.
```

Comment:
I think we should only support mime types, not file extensions.  

I'd be against using wildcards in the manner suggested here. Let's just stick to well-formed/valid MIME Types per:
https://mimesniff.spec.whatwg.org 

Additionally, we should have a list of allowed types that we all agree on, otherwise this API has the potential to becomes a fingerprinting vector.

Reply:
I agree this needs more details, also for web developers who should have some assurances about the capabilities.